nginx.conf

```diff
@@ -0,0 +1,78 @@
+
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    # gzip
+    gzip on;
+    gzip_min_length 1000;
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_types text/plain text/html text/css text/xml application/json application/javascript application/xml+rss application/atom+xml image/svg+xml image/x-icon;
+
+    resolver 127.0.0.11 valid=5s;
+
+    server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+        root /var/static;
+        index index.html;
+
+        location /flags/ {
```

Comment:
We should actually prefix all API routes with `/api/v1` in `api.py`: this way we can easily create a new route version with incompatible parameters if needed.
And you can do exactly as open-prices here in nginx.conf.